extension/src/popup/ducks/transactionSubmission.ts

```diff
@@ -99,6 +104,21 @@ export const getDestinationBalances = createAsyncThunk<
   }
 });
 
+export const getAssetIcons = createAsyncThunk<
+  AssetIcons,
+  { balances: Balances; networkDetails: NetworkDetails },
+  { rejectValue: ErrorMessage }
+>(
+  "auth/getAssetIcons",
+  ({
+    balances,
+    networkDetails,
+  }: {
+    balances: Balances;
+    networkDetails: NetworkDetails;
+  }) => getAssetIconsService({ balances, networkDetails }),
+);
```

Comment:
I'm adding `assetIcons` to the redux store because we always fetch all these icons when we land on `Account`, and we need them for subsequent views like Manage Assets and Send Payment. I figure we can save ourselves reaching back out for toml files by saving the url's in redux for reuse

Ideally, this will hold an asset's home domain, as well, at some point